src/video/SDL_video.c

```diff
@@ -358,6 +358,13 @@ static void SDL_DestroyWindowTexture(SDL_VideoDevice *unused, SDL_Window *window
     SDL_free(data);
 }
 
+static double get_refresh_rate(const SDL_DisplayMode *dm) {
```

Comment:
Is double precision required or would a float suffice?

Reply:
double sounds better since we use num/den for better precesion